zerver/management/commands/deliver_email.py

```diff
@@ -43,7 +44,7 @@ def handle(self, *args: Any, **options: Any) -> None:
         if settings.EMAIL_DELIVERER_DISABLED:
             sleep_forever()
 
-        with lockfile("/tmp/zulip_email_deliver.lockfile"):
+        with lockfile(os.path.join(settings.DEPLOY_ROOT, "email_deliverer.lockfile")):
```

Comment:
I think this probably has the same multiple deployments problem?  Less clear.

Reply:
I think what I actually want to do wit this file is just get rid of the lock; my believe is that it is useless.

Reply:
Removed.